src/entryPoints/HyperSwitchApp.res

```diff
@@ -243,9 +243,13 @@ let make = () => {
                         }}
                       />
                     </div>
-                    <RenderIf condition=isLiveUsersCounterEnabled>
-                      <ActiveUserCounter />
-                    </RenderIf>
+                    {switch url.path->urlPath {
+                    | list{"home"} =>
+                      <RenderIf condition=isLiveUsersCounterEnabled>
+                        <ActivePaymentsCounter />
+                      </RenderIf>
+                    | _ => React.null
+                    }}
```

Comment:
Do we need extra switch here and rather than that can we  add it here itself ? 
```
{switch url.path->urlPath {
 | list{"home"} => featureFlagDetails.quickStart ? <HomeV2 /> : <Home />
```

Reply:
Yes.